src/WindowsTracing/EtwEventTypes.h

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef WINDOWS_TRACING_EVENT_TYPES_H_
+#define WINDOWS_TRACING_EVENT_TYPES_H_
+
+#include <cstdint>
+
+namespace orbit_windows_tracing::etw {
+
+// ETW event types grouped by event class. Note that an event type name is of the form
+// "[parent class]_[event class]". The parent class is associated with a guid that is used
+// to dispatch events to a particular callback for processing.
+// See https://docs.microsoft.com/en-us/windows/win32/etw/event-tracing-mof-classes.
+
+// https://docs.microsoft.com/en-us/windows/win32/etw/thread-typegroup1
+namespace Thread_TypeGroup1 {
```

Comment:
So again, we decided not to use nested namespaces and I'd like to avoid exceptions (even more as `orbit_windows_tracing::etw` could be mistaken with a possible `etw` and the inner ones can't follow snake_case).

`struct`s are fine for grouping, just I was confused by the homonymy. What about just a suffix? For example:
`struct Thread_TypeGroup1_Constants`, or `struct Thread_TypeGroup1_Ids`, or `struct Thread_TypeGroup1_EventTypeIds`, ...?

Reply:
Went for flat list of constants.